examples/todo-list/src/controllers/author.controller.ts

```diff
@@ -0,0 +1,122 @@
+import {
+  Count,
+  CountSchema,
+  Filter,
+  repository,
+  Where,
+} from '@loopback/repository';
+import {
+  post,
+  param,
+  get,
+  getFilterSchemaFor,
+  getWhereSchemaFor,
+  patch,
+  del,
+  requestBody,
+} from '@loopback/rest';
+import {Author} from '../models';
+import {AuthorRepository} from '../repositories';
+
+export class AuthorController {
```

Comment:
Since every author belongs to a single todo, does it actually make sense to expose authors directly? Isn't it better to expose only the has-one relation API?

Just an idea to consider.

Reply:
Yeah I was contempleting that as well and I think it makes sense just to expose the `has-one` relation APIs.